packages/avs-bot/src/avs.ts

```diff
@@ -0,0 +1,195 @@
+    wCall.init(avsEnv);
+    wCall.setUserMediaHandler(this.mediaHandler);
+    wCall.setMediaStreamHandler(this.onIncomingStream);
+    setInterval(() => wCall.poll(), 500);
+    return wCall;
+  }
+
+  onIncomingStream = (
+    convid: string,
+    remote_userid: string,
+    remote_clientid: string,
+    streams: readonly MediaStream[],
+  ): void => {};
+
+  mediaHandler = async (): Promise<MediaStream> => {
+    const mediaStream = this.mediaStream;
+    const track = this.audioSource.createTrack();
+    mediaStream.addTrack(track);
+    return mediaStream;
+  };
+
+  createWUser(wCall: Wcall, selfUserId: string, selfClientId: string): number {
+    /* cspell:disable */
+    const wUser = wCall.create(
+      selfUserId,
+      selfClientId,
+      () => {},
+      this.onSendCallMessage,
+      this.sendSFTRequest,
+      this.onIncomingCall,
+      () => {},
+      () => {},
+      () => {},
+      this.callClosed,
+      () => {},
+      () => 0,
+      () => {},
+      () => {},
+    );
+
+    wCall.setParticipantChangedHandler(wUser, this.updateCallParticipants);
+    return wUser;
+  }
+
+  private readonly sendSFTRequest = (
+    context: number,
+    url: string,
+    data: string,
+    dataLength: number,
+    _: number,
+  ): number => {
+    console.log('\n sendSFTRequest \n');
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    (async () => {
+      const response = await axios.post(url, data);
+
+      const {status, data: axiosData} = response;
+      const jsonData = JSON.stringify(axiosData);
+      this.wCall!.sftResp(this.wUser!, status, jsonData, jsonData.length, context);
+    })();
+    return 0;
+  };
+
+  onSendCallMessage = (
+    ctx: number,
+    conversationId: string,
+    selfUserId: string,
+    selfClientId: string,
+    userid_dest: string | undefined,
+    clientid_dest: string | undefined,
+    data: string,
+    len: number,
+    trans: number,
+    arg: number,
+  ): number => {
+    void (async () => {
+      try {
+        const callPayload = this.account!.service!.conversation.messageBuilder.createCall(conversationId, data);
+        await this.account!.service!.conversation.send(callPayload);
+      } catch (error) {
+        console.error('onSendCallMessage error', error);
+      }
+    })();
+
+    return 0;
+  };
+
+  updateCallParticipants = (conversationId: string, membersJson: string) => {};
+
+  private readonly onIncomingCall = (
+    conversationId: string,
+    timestamp: number,
+    userId: string,
+    clientId: string,
+    hasVideo: number,
+    shouldRing: number,
+  ) => {
+    const call: Call = {
+      clientId,
+      conversationId,
+      hasVideo,
+      shouldRing,
+      timestamp,
+      userId,
+    };
+    this.storeCall(call);
+  };
+
+  private readonly callClosed = (reason: REASON, conversationId: string) => {
+    this.audioSource.stop();
+    this.removeCall(conversationId);
+  };
+
+  storeCall(call: Call) {
```

Comment:
Can you check which functions in this file can be private it seems there are some.

Reply:
https://github.com/wireapp/wire-web-packages/pull/3070/commits/16edff26574f9ab4f1e8b56685ea1dabf52dedd7